README.md

```diff
@@ -1,11 +1,11 @@
-# mattermost-github-integration
+# Mattermost-Github-integration
 
 This role sets up a mattermost github integration service which posts changes of an organization or repository to a mattermost channel.
```

Comment:
I think @dasJ actually meant the sentence, not the headline

Reply:
fixed